### DISCUSSION TITLE
Icon

### DISCUSSION THREAD

**Discussion by otalado** [pawelsalawa/sqlitestudio#4493] [Ideas]
Hi,
I just downloaded SQLiteStudio to Debian 11 (x64) and I like it. Of course one of the first things I've done was to create a desktop shortcut entry and start  menu entry for it. I've spent quite some time finding an adequate icon for that. 
Just an idea: it would be really minor (**and really nice and handy**) thing to do, if you include an SQLiteStudio icon in the distribution.

**Comment by pawelsalawa** [comment]
I think #4264 will solve your problem, right?
